sorc/link_workflow.sh

```diff
@@ -207,29 +247,29 @@ for workflowexec in fbwndgfs gfs_bufr regrid_nemsio supvit syndat_getjtbul \
   $LINK ../sorc/install/bin/${workflowexec}.x $workflowexec
 done
 for workflowexec in enkf_chgres_recenter.x enkf_chgres_recenter_nc.x fv3nc2nemsio.x \
-    tave.x vint.x ; do
+    tave.x vint.x reg2grb2.x ; do
   [[ -s $workflowexec ]] && rm -f $workflowexec
   $LINK ../sorc/install/bin/$workflowexec .
 done
 
-[[ -s global_fv3gfs.x ]] && rm -f global_fv3gfs.x
-$LINK ../sorc/fv3gfs.fd/NEMS/exe/global_fv3gfs.x .
+[[ -s ufs_model ]] && rm -f ufs_model
+$LINK ../sorc/ufs_model.fd/build/ufs_model .
 
 [[ -s gfs_ncep_post ]] && rm -f gfs_ncep_post
 $LINK ../sorc/gfs_post.fd/exec/upp.x gfs_ncep_post
 
 if [ -d ${pwd}/gfs_wafs.fd ]; then 
     for wafsexe in \
-          wafs_awc_wafavn.x  wafs_blending.x  wafs_blending_0p25.x \
-          wafs_cnvgrib2.x  wafs_gcip.x  wafs_grib2_0p25.x \
-          wafs_makewafs.x  wafs_setmissing.x ; do
+          wafs_awc_wafavn  wafs_blending  wafs_blending_0p25 \
+          wafs_cnvgrib2  wafs_gcip  wafs_grib2_0p25 \
+          wafs_makewafs  wafs_setmissing; do
```

Comment:
The hash of WAFS in the develop branch produces execs with ".x" in the name. Does the linking work with this change?

Reply:
I'm going to guess no. I didn't pick up on this in testing because no WAFS links are produced at all, so there were no broken links to see.